### PR TITLE
Adds the u-ION silencer to rnd, replacing ecrossbow. Ecrossbow up to syndicate 5

### PR DESCRIPTION
**Pull request by Qwertytoforty**
## What Does This PR Do

**EBOW RND CHANGES**

Ecrossbow now needs syndicate 5.
In order to reach this level, the following items (or 2 equivalent items/0 must be deconstructed.

2 cham projectors
syndicate bomb core
powersink
syndicate bomb beacon
hacked ai core
dsword
2 chainsaws
2 adrenilin
uplink implant
2 contractor uplinks
2 cham thermals
syndie ebow
mindslave

All of these purchases cost 10 or more TC. This means that chances are, this is not a better deal than buying a syndicate crossbow.

One exception was the stechkin (4 tc) pistol.

Its tech value has been lowered to 3

**u-ION silencer**

The new RND weapon to replace the ecrossbows place.

"u-ION Silencer"
"Nanotrasen's take on silenced weapons. A quiet lethal disabler, designed to make death look like a natural cause."

Think of it as a silenced, yet lethal, disabler.

16 shots.

Each shot is 30 stamina, and 15 oxygen damage.

It is silenced. Sounds of shooting is much quieter compared to a normal laser gun, similarly to how a silenced stech is quiet compared to normal.

Requires a disabler to be converted.

## Why It's Good For The Game

Hi, me again, touching the ebow.

The rnd ebow is in a tough place. Laser or KA, it is still incredibly cheap compared to paying for the syndicate version. You are saving 8 TC. That is a lot.

As such, it should be moved to a higher illegals level, that costs more TC to buy. When we roll around to reworked rnd, it can be made that only items worth 10+ TC can be used to unlock the tree.

However, something should take it's place. We need a weapon, NT in design, yet with syndicate heavily incorporated into it.

As such, we get the u-ION Silencer. An energy weapon, built for silent killing, disguised as a heart attack / suffocation. 

**Compared to the RND EBOW**

Not infinite ammo, the biggest issue, and as such requires recharger visits.

Doesn't 2 hit permastun.

**Compared to other RND weapons.**

Can't be used to kill everything, only humans.

A silenced energy weapon, which fits the syndidcate theming.

Actually useful as an assassination weapon, as with enough o2, person will be unable to call for help (Potentially problematic, although thematic, though opening with ion rifle has same effect)

Doesn't deal brute / burn, deals o2, which heals naturally, is easy to heal with medicine, and some healing viruses will negate.

## Images of changes

image

## Testing

Ensure weapon sounded quiet.
Ensure silenced chat logs worked.
Ensured it did expected damage.
Ensured it could be printed at right tech levels in rnd.

## Changelog
:cl:
add: u-ION Silencer, a hybrid disabler and lethal silenced energy weapon, available with some syndicate technology in RND.
tweak: Syndicate pistol gives syndicate 3 vs syndicate 4.
tweak: RND ebow now requires illegals 5
/:cl:

### CODE REVIEW

**Comment by roarbark** [comment]
this is a cool design and ebow needs this bump up.

though, as a weapon what role is this supposed to fulfil? its very obviously just a weapon for science antags to use, because like, when would you want to sacrifice 4 shots for 15 oxygen damage each as sec, when you could just bust out a lasergun.

so the use cases seem to be 1) killing your targets while muting them as efficiently as possible and 2) killing security officers that are like, alone by themselves in a hallway.

the difference to opening with ions is that you need a lasergun to even craft that, then a disabler to have the same amount of stunning power and then something to kill them, while making very loud and obvious noises and giving more time for them to run or fight back.

i love the idea of a disabler sidegrade but to me especially use case one seems really really minmaxy to get your greentext without anyone even knowing. 

i dont think this is gonna be actively harmful because to build this you need to have already stripped a seccie or like pulled a disabler crate under the noses of cargo, but still, maybe im missing something but i don't know, i'm skeptical.

File: code/modules/crafting/recipes.dm [hunk 1]
```diff
@@ -218,6 +218,21 @@
 	..()
 	blacklist += subtypesof(/obj/item/gun/energy/laser)
 
+/datum/crafting_recipe/silencer
+	name = "u-ION Silencer"
+	tools = list(TOOL_SCREWDRIVER, TOOL_WIRECUTTER)
+	result = list(/obj/item/gun/energy/disabler/silencer)
+	reqs = list(/obj/item/gun/energy/gun = 1,
+				/obj/item/stack/cable_coil = 5,
+				/obj/item/weaponcrafting/gunkit/silencer = 1)
```
Comment (hal9000PR) [inline code/modules/crafting/recipes.dm:227]: (design) I feel like making it out of a disabler would make more sense.

File: code/modules/projectiles/guns/energy/stun.dm [hunk 2]
```diff
@@ -67,3 +67,13 @@
 /obj/item/gun/energy/disabler/cyborg/newshot()
 	..()
 	robocharge()
+
+/obj/item/gun/energy/disabler/silencer
+	name = "u-ION Silencer"
```
Comment (hal9000PR) [inline code/modules/projectiles/guns/energy/stun.dm:72]: is this name supposed to be a reference to something? its not an ion gun, seems confusing

**Review by hal9000PR — DISMISSED** [review]

File: code/modules/crafting/guncrafting.dm [hunk 3]
```diff
@@ -75,6 +75,11 @@
 	desc = "A suitcase containing the necessary gun parts to transform a standard laser gun into a plasma pistol. Wort, wort, wort!"
 	origin_tech = "combat=4;magnets=4;powerstorage=3"
 
+/obj/item/weaponcrafting/gunkit/silencer
```
[the remark this answers is not in the document]
Comment (Qwertytoforty) [reply code/modules/crafting/guncrafting.dm:78]: Fair, can change it

File: code/modules/crafting/recipes.dm (hunk 1 above)
Comment (Qwertytoforty) [reply code/modules/crafting/recipes.dm:227]: Up for trying that

File: code/modules/projectiles/guns/energy/stun.dm (hunk 2 above)
Comment (Qwertytoforty) [reply code/modules/projectiles/guns/energy/stun.dm:72]: Joke on union silencer,  NT having a way to "naturally" kill off their competition seems like something they would do, however ION in it is problematic. Open for other name suggestions

File: code/modules/crafting/guncrafting.dm (hunk 3 above)
[the remark this answers is not in the document]
Comment (Qwertytoforty) [reply code/modules/crafting/guncrafting.dm:78]: Changed!

File: code/modules/crafting/recipes.dm (hunk 1 above)
Comment (Qwertytoforty) [reply code/modules/crafting/recipes.dm:227]: Done!

**Review by Sirryan2002 — COMMENTED** [review]
code looks good, the only thing is farie's spell check review that needs to be fixed

**Review by farie82 — APPROVED** [review]

File: code/modules/projectiles/guns/energy/stun.dm (hunk 2 above)
Comment (farie82) [reply code/modules/projectiles/guns/energy/stun.dm:72]: @hal9000PR can you say if this comment is resolved or not? Code looks good as is

Comment (hal9000PR) [reply code/modules/projectiles/guns/energy/stun.dm:72]: @farie82 yeah i don't have any better ideas.

**Pull request merged**